src/algebra/order/monoid_lemmas.lean

```diff
@@ -213,9 +320,12 @@ end has_lt
 section preorder
 variable [preorder α]
 
+-- Lemmas in the form of `b ≤ c → a ≤ 1 → b * a ≤ c`,
+-- which assume left covariance.
+
 @[to_additive]
 lemma mul_le_of_le_of_le_one [covariant_class α α (*) (≤)]
-  {a b c : α} (hbc : b ≤ c) (ha : a ≤ 1) : b * a ≤ c :=
```

Comment:
I'd argue it's better to keep these variables here, as it ensures they appear after the typeclass arguments

Reply:
(same throughout)